tests/MethodTest.php

```diff
@@ -48,6 +49,33 @@ public function testOutput()
         $this->assertEquals(true, $method->shouldReturn());
     }
 
+    /**
+     * Test the output of a class
+     */
+    public function testEloquentBuilderOutput()
+    {
+        $reflectionClass = new \ReflectionClass(Builder::class);
+        $reflectionMethod = $reflectionClass->getMethod('with');
+
+        $method = new Method($reflectionMethod, 'Builder', $reflectionClass);
+
+        $output = '/**
+ * Set the relationships that should be eager loaded.
+ *
+ * @param mixed $relations
+ * @return \Illuminate\Database\Eloquent\Builder|static 
+ * @static 
+ */';
+        $this->assertEquals($output, $method->getDocComment(''));
+        $this->assertEquals('with', $method->getName());
+        $this->assertEquals('\\'.Builder::class, $method->getDeclaringClass());
+        $this->assertEquals('$relations', $method->getParams(true));
+        $this->assertEquals(['$relations'], $method->getParams(false));
+        $this->assertEquals('$relations', $method->getParamsWithDefault(true));
+        $this->assertEquals(['$relations'], $method->getParamsWithDefault(false));
+        $this->assertEquals(true, $method->shouldReturn());
```

Comment:
I would suggest to use `assertSame` everywhere, don't see a reason for not using the strictes possible check here

Reply:
@mfn Yes I use `assertSame` on all my projects this was just a copy and paste of the other output test. I have changed the whole test to now use `assertSame` for consistency.

Reply:
I know what you mean 😄 

I always try to be bit of boy scout here and trying to leave the place in a better shape I found it… can't help this OCD 🤷‍♀️